src/main/java/DrawCard/DrawCardInputData.java

```diff
@@ -0,0 +1,38 @@
+package DrawCard;
+
+import MainEntities.Deck;
+
+public class DrawCardInputData {
+    // boolean confirm;
+    private String playerName;
+
+    /**
+     *
+     * @param playerName the name of the player who has to draw a Card.
+     */
+    private Deck deckType;
+
+    /**
+     *
+     * @param deckType The type of Deck that the player has to draw on.
+     */
+
+    public DrawCardInputData(String playerName, Deck deckType) {
```

Comment:
Your use case does not need any input data. Once a player lands on a tile, the advance use case will determine what tile it is and call the appropriate use case (in jail, draw card, pay rent, etc.).